### ISSUE TITLE
global bot notification settings shouldn't overwrite room notification settings

### ISSUE THREAD

**Issue by schiessle** [vector-im/riot-web#10399]
I have a few rooms where a bot post updated of RSS feeds, for this updates I would like to get notifications. Therefore I set "Messages sent by bot" to "noisy" in the account settings.

But now I get notified for all bot messages, also in rooms where the room notifications are set to "mute" or "mentions only".

I would expect that room settings overwrite the account wide settings so that notifications for bot messages are only triggers where the room notification setting is set to "all messages".

**Comment by jryans** [comment]
Thanks for the feedback. Notifications are quite complex with many interrelated conditions. I agree this seems like something to improve.

**Comment by downeymj** [comment]
I agree. I would love to take advantage of notifications from certain bots like Go-NEB (RSS) etc., but right now with the "one size fits all" bot notification setting, I can only get notifications from all bots in all rooms that I'm in, or no notifications at all. Bot notifications should be a per-room toggle, not global -- and not only here in element-web but also in Android, etc.
